crates/biome_js_analyze/tests/specs/nursery/noDefaultExport/invalid.json.snap

```diff
@@ -0,0 +1,119 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: invalid.json
+---
+# Input
+```js
+export default function f() {};
+```
+
+# Diagnostics
+```
+invalid.json:1:8 lint/nursery/noDefaultExport ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Use a named export instead.
+  
+  > 1 │ export default function f() {};
```

Comment:
The range should highlight `export default` or `default`. It should not point to the whole function.